studio/components/interfaces/BillingV2/Usage/Infrastructure.tsx

```diff
@@ -90,6 +90,10 @@ const Infrastructure = ({
     0
   )
 
+  const latestIoBudgetConsumption = ioBudgetData?.data.slice(-1)
```

Comment:
If we use optional chaining, we need to use it for every chained call, otherwise follow-up calls may fail, in this case `.slice`